packages/mock-block-dock/package.json

```diff
@@ -35,6 +35,7 @@
     "run-dev-server": "cross-env NODE_ENV=development webpack-dev-server --config dev/webpack.config.js"
   },
   "dependencies": {
+    "@blockprotocol/graph": "0.0.2",
     "blockprotocol": "0.0.10",
```

Comment:
Don't we have to use `@blockprotocol/core` instead of `blockprotocol` here?

Reply:
Good spot - this is actually unused. Removed in https://github.com/blockprotocol/blockprotocol/pull/326/commits/175e1113a3122f39ee714b485f7db133a0436fe5